its/ruling/src/test/resources/expected/ruby/ruby-S1314.json

```diff
@@ -0,0 +1,180 @@
+652,
+714,
+740,
+740,
+],
+'ruby-project:sources/ruby/rails/activerecord/test/cases/date_test.rb':[
+8,
+],
+'ruby-project:sources/ruby/rails/activerecord/test/cases/date_time_precision_test.rb':[
+83,
+],
+'ruby-project:sources/ruby/rails/activerecord/test/cases/time_precision_test.rb':[
+77,
+],
+'ruby-project:sources/ruby/rails/activesupport/test/core_ext/date_and_time_behavior.rb':[
+136,
+269,
+270,
+271,
+272,
+273,
+274,
+275,
+277,
+278,
+279,
+280,
+281,
+288,
+289,
+290,
+291,
+292,
+293,
+294,
+303,
+304,
+305,
+306,
+316,
+317,
+],
+'ruby-project:sources/ruby/rails/activesupport/test/core_ext/date_ext_test.rb':[
+93,
+],
+'ruby-project:sources/ruby/rails/activesupport/test/core_ext/duration_test.rb':[
+178,
+],
+'ruby-project:sources/ruby/rails/activesupport/test/core_ext/file_test.rb':[
+30,
+35,
+43,
+],
+'ruby-project:sources/ruby/rails/activesupport/test/core_ext/range_ext_test.rb':[
+98,
+98,
+104,
+],
+'ruby-project:sources/ruby/rails/activesupport/test/core_ext/time_ext_test.rb':[
+407,
+407,
+761,
+],
+'ruby-project:sources/ruby/rails/activesupport/test/core_ext/time_with_zone_test.rb':[
+296,
+303,
+],
+'ruby-project:sources/ruby/rails/activesupport/test/json/decoding_test.rb':[
+39,
+],
+'ruby-project:sources/ruby/rails/activesupport/test/json/encoding_test.rb':[
+307,
+307,
+],
+'ruby-project:sources/ruby/rails/activesupport/test/notifications_test.rb':[
+300,
+300,
+],
+'ruby-project:sources/ruby/rails/activesupport/test/time_travel_test.rb':[
+46,
+46,
+60,
+60,
+77,
+77,
+96,
+96,
+97,
+97,
+111,
+111,
+112,
+112,
+126,
+126,
+127,
+127,
+],
+'ruby-project:sources/ruby/rails/activesupport/test/xml_mini_test.rb':[
+132,
+242,
+254,
+254,
+256,
+257,
+267,
+278,
+289,
+310,
+],
+'ruby-project:sources/ruby/rails/railties/lib/rails/generators/rails/app/app_generator.rb':[
+93,
+],
+'ruby-project:sources/ruby/rails/railties/lib/rails/generators/rails/encryption_key_file/encryption_key_file_generator.rb':[
+30,
+],
+'ruby-project:sources/ruby/rails/railties/lib/rails/generators/rails/plugin/plugin_generator.rb':[
+162,
+],
+'ruby-project:sources/ruby/vagrant/lib/vagrant/environment.rb':[
+1002,
+],
+'ruby-project:sources/ruby/vagrant/lib/vagrant/plugin/state_file.rb':[
+106,
+],
+'ruby-project:sources/ruby/vagrant/lib/vagrant/util/ssh.rb':[
+43,
+],
+'ruby-project:sources/ruby/vagrant/plugins/hosts/bsd/cap/ssh.rb':[
+11,
+],
+'ruby-project:sources/ruby/vagrant/plugins/hosts/linux/cap/ssh.rb':[
+11,
+],
+'ruby-project:sources/ruby/vagrant/test/unit/plugins/hosts/bsd/cap/ssh_test.rb':[
+12,
+],
+'ruby-project:sources/ruby/vagrant/test/unit/plugins/hosts/linux/cap/nfs_test.rb':[
+258,
+],
+'ruby-project:sources/ruby/vagrant/test/unit/plugins/hosts/linux/cap/ssh_test.rb':[
+12,
+],
+'ruby-project:sources/ruby/vagrant/test/unit/vagrant/util/ssh_test.rb':[
+15,
+24,
+27,
+],
+'ruby-project:sources/ruby/vagrant/test/unit/vagrant/util/which_test.rb':[
```

Comment:
Looking at results here, I would add two exceptions to the rule - when value is in range 1 - 7, because the result is same regardless the base. And for values related to unix permissions 0000 - 0777 , this is often used and completely expected imo